fees/ether-fi/index.ts

```diff
@@ -31,8 +98,31 @@ const getPayoutDetails = async (options, target) => {
   return [asset_eth, rate_eth];
 };
 
+const getSsvRevenue = async (options: FetchOptions) => {
+  const logs = await options.getLogs({
```

Comment:
sorry, missed to check this, can split it into two log fetches, one with fromFilter and other with toFilter like this https://github.com/DefiLlama/dimension-adapters/blob/master/fees/ethena.ts#L38, this way we avoid pulling only relevant logs and avoid load on our indexer.

Reply:
okay, I will fix this part myself